x-pack/plugins/apm/server/lib/services/get_services.ts

```diff
@@ -31,23 +32,8 @@ export async function getServices(
   const { start, end, esFilterQuery, client, config } = setup;
 
   const filter: ESFilter[] = [
-    {
-      bool: {
-        should: [
-          { term: { [PROCESSOR_EVENT]: 'transaction' } },
-          { term: { [PROCESSOR_EVENT]: 'error' } }
-        ]
-      }
-    },
-    {
-      range: {
-        '@timestamp': {
-          gte: start,
-          lte: end,
-          format: 'epoch_millis'
-        }
-      }
-    }
+    { terms: { [PROCESSOR_EVENT]: ['transaction', 'error'] } },
```

Comment:
Note: possible to specify several terms at once (thanks @ogupte for showing me this). This means we can get rid of the `should` clause.